pkg/scheduler/apis/config/v1/defaults_test.go

```diff
@@ -45,8 +46,8 @@ var pluginConfigs = []configv1.PluginConfig{
 					Kind:       "DefaultPreemptionArgs",
 					APIVersion: "kubescheduler.config.k8s.io/v1",
 				},
-				MinCandidateNodesPercentage: pointer.Int32Ptr(10),
-				MinCandidateNodesAbsolute:   pointer.Int32Ptr(100),
+				MinCandidateNodesPercentage: pointer.Int32(10),
```

Comment:
done

Reply:
Bump. My idea is to use `Int32` function for update/addition changes introduced in this PR, and leave existing bits as-is.

(So if you want to backport internally, it would be much easier)